src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -48,7 +49,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
-    ## This label is important as it is required for the Prometheus Agent to discover it.
+    ## This label is important as it is required for the metrics agent to discover it.
     ## The team name should be the name of your internal team.
     application.giantswarm.io/team: my-team
```

Comment:
That's wrong though :D We should only show the new label right?

Reply:
Oh alright. So I think we should get rid of all the vintage stuff here and anything related to prometheus agent and just explain the new one with a link to the tenancy documentation.

I would emphasize that the tenant needs to be references in an existing grafana organization, the same way we did it for logs

Reply:
Yeah you're right.